hack/app-sre/operator_role_binding.yaml

```diff
@@ -0,0 +1,21 @@
+# HACK: This is the same as config/operator/operator_role_binding.yaml *except*
+# that the subjects[0].namespace is hardcoded. This is because:
+# - `kustomize set namespace` will *set* the subjects[].namespace if absent,
```

Comment:
Yes, sorry, I didn't explain it very well (because it's confusing as hell).

I can't hardcode it into config/operator/operator_role_binding.yaml because then `make deploy` won't override it properly according to `HIVE_NS` for regular deploys and e2e*.

I can't use kustomize to set it during the app-sre deploy because the app-sre deploy just looks at a static file. (It supports parameters; but then I'd have to parameterize it for both deploy types. That's another option here; I don't think one is better than the other.)

And I can't use kustomize to set it as part of `make build-app-sre-template' because [Maor told me not to](https://github.com/openshift/hive/pull/1486#discussion_r683158181).

Reply:
that makes sense. 👍🏽